### PR TITLE
Release CLI tool on tag push

### LINKED ISSUE #123: Add CLI jar to release assets on tag
Create new GitHub action to build [CLI RPM tool](https://github.com/artipie/rpm-adapter#cli) on release GitHub event and upload target `target/rpm-adapter.jar` to release assets similar to https://github.com/artipie/white-paper/blob/master/.github/workflows/build-latex.yml#L28

### PR DESCRIPTION
**Pull request by Sammers21**
Closes #123

### CODE REVIEW

**Comment by g4s8** [comment]
@0crat status

File: .github/workflows/release-cli.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  build:
+    name: Build and Publish CLI Tool
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
```
Comment (g4s8) [inline .github/workflows/release-cli.yml:11]: @Sammers21 let's use `actions/checkout@v2` it's faster and has several bug fixes

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/artipie/rpm-adapter/issues/127#issuecomment-615076557))

@g4s8 This is what I know about this job in CT2E6TK9B, as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#127](https://github.com/artipie/rpm-adapter/issues/127) is [in scope](http://www.zerocracy.com/policy.html#14) for 9hr
  * The role is `REV`
  * The job is not assigned to anyone
  * The [budget](http://www.zerocracy.com/policy.html#4) is not set yet
  * These users are banned and won't be assigned:
    * @sammers21[/z](https://www.0crat.com/u/sammers21): This user reported the ticket
  * Job [footprint](https://www.0crat.com/footprint/CT2E6TK9B?q=%7Bjob%3A%27gh%3Aartipie%2Frpm-adapter%23127%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by 0crat** [comment]
This pull request [#127](https://github.com/artipie/rpm-adapter/issues/127) is assigned to @g4s8[/z](https://www.0crat.com/u/g4s8), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/7fc8cf43-36a3-4ae7-8685-5cd273de7275); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

**Comment by 0crat** [comment]
Job was finished in 0 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Pull request merged**